docker-compose.yml

```diff
@@ -2,10 +2,10 @@ version: '3'
 services:
   volttron:
     hostname: vc
-    image: volttron/volttron
+    image: bonicim/volttron-docker-dev:test
```

Comment:
I built the image using the Dockerfile in this repo and pushed to my personal dockerhub account. For now, using this image to run docker-compose.